### PR TITLE
Add migration detail for Besu users and support language

### PR DESCRIPTION
**Pull request by vmichalik**
## PR description
Adds clarifying detail to explain impact on Besu and language on support

### CODE REVIEW

File: docs/Tutorials/Migrating-from-Orion-to-Tessera.md [hunk 1]
```diff
@@ -7,16 +7,20 @@ description: Migrate your Orion configuration and data to Tessera.
 
 ## Migration Process
 
+We recommend use of Tessera as a drop-in replacement private transaction manager in place of Orion.
+As Tessera supports the same endpoints and functionality, nothing needs to be changed in your Besu deployment to migrate.
+However, short downtime is required to migrate the private state database, transaction manager configuration and associated key files.
```
Comment (nmvalera) [inline docs/Tutorials/Migrating-from-Orion-to-Tessera.md]: Maybe "short downtime" is misleading of the actual requirements on Ops teams to perform the migration.

Maybe we could precise that
-> all private nodes (Besu+Priv Tx Manager) in the network need to go down at the same time to perform the migration
-> non-private nodes can remain up
-> [optional] Tessera and Orion can not communicate with each other (thus the requirement for network interuption)

Comment (vmichalik) [reply docs/Tutorials/Migrating-from-Orion-to-Tessera.md]: Thank you. I made some edits to address your suggestions, let me know what you think.

File: docs/Tutorials/Migrating-from-Orion-to-Tessera.md [hunk 2]
```diff
@@ -193,3 +198,9 @@ You must specify the following options in order to run the migration tool:
 `password`= Target Tessera database password
 
 `url`= Target Tessera database JDBC connection string
+
+# Support
```
Comment (NicolasMassart) [inline docs/Tutorials/Migrating-from-Orion-to-Tessera.md]: Maybe be clear about the support type so we don't have to refuse requests from non customers who will be disappointed.
```suggestion
# Enterprise support
```

**Review by NicolasMassart — APPROVED** [review]
Looks good to me, just a word change suggestion.

File: docs/Tutorials/Migrating-from-Orion-to-Tessera.md (hunk 1 above)
Comment (nmvalera) [reply docs/Tutorials/Migrating-from-Orion-to-Tessera.md]: Looks good, thanks

Maybe in the steps you also want to highlight it in the steps

```md
1. Shut down all privacy-enabled nodes in the network (non-privacy-enabled nodes can remain active)
...
1. Restart  all privacy-enabled nodes in the network
```

**Pull request merged**